Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -8112,15 +8112,16 @@ void WebPageProxy::resetStateAfterProcessTermination(ProcessTerminationReason re
     pageClient().removeAllPDFHUDs();
 #endif
 
-    // For bringup of process swapping, NavigationSwap termination will not go out to clients.
-    // If it does *during* process swapping, and the client triggers a reload, that causes bizarre WebKit re-entry.
-    // FIXME: This might have to change
-    if (reason != ProcessTerminationReason::NavigationSwap)
+    if (reason != ProcessTerminationReason::NavigationSwap) {
+        // For bringup of process swapping, NavigationSwap termination will not go out to clients.
```

Comment:
NIT: does this comment need to be moved?

Reply:
I guess this comment line could still go outside. I just wanted to make sure that the FIXME and its context stayed with the target line so that it wasn't mistakenly applied to the WebDriver stuff in the future as well.